src/check-for-spam.js

```diff
@@ -0,0 +1,41 @@
+// content refers to the contents of the blog post (article) sent by feedparser
+module.exports = function (content) {
+  // Minimum words set to 20, can be changed
+  const MINWORDS = 20;
+
+  // Functions
+  function removeTags(str) { // Removes HTML tags from a string, leaving only plaintext
+    if (str != null && str !== '') {
+      return str.toString().replace(/<[^>]*>/g, '');
+    }
+    return '';
+  }
+
+  function getWordCount(str) { // Counts number of words in string
+    return str.split(' ').length;
+  }
+
+  function getCapitalLetters(str) { // Counts number of capital letters in string
+    return str.replace(/[^A-Z]/g, '').length;
+  }
+
+  function getAllLetters(str) { // Counts number of alphabetical characters in string
+    return str.replace(/[^A-Za-z]/g, '').length;
+  }
+
+  // The main body of the blog post with all HTML tags removed
+  const noTagsDesc = removeTags(content.description);
+
+  // If the title is empty, post is considered spam
+  if (content.title != null) {
```

Comment:
change to 
```js
if(content.hasOwnProperty(title)){...}
```
checkout the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) for details.

Reply:
I can't use that cause content always has the property title, its just that if the title of the blog is blank, content.title it returns null.